packages/terra-worklist-data-grid/src/WorklistDataGrid.module.scss

```diff
@@ -4,9 +4,9 @@
 
 :local {
   .worklist-data-grid [tabIndex='0']:focus {
-    background-color: var(--terra-worklist-data-grid-cell-focus-background-color, #cbe7fa);
-    box-shadow: var(--terra-worklist-data-grid-cell-focus-box-shadow, inset 0 0 0 1px rgba(38, 162, 229, 0.5), inset 0 0 0 4px rgba(76, 178, 233, 0.2));
-    outline: none;
+    outline: var(--terra-worklist-data-grid-cell-focus-outline, 2px dashed #000);
+    outline-offset: var(--terra-data-grid-cell-focus-outline-offset, -2px);
+    z-index: 1;
```

Comment:
I would not expect us to have to set the z-index for the implementation.

Reply:
Removed